packages/@aws-cdk/aws-redshift/test/database-query-provider/table.test.ts

```diff
@@ -92,7 +92,7 @@ describe('create', () => {
     await manageTable(newResourceProperties, event);
 
     expect(mockExecuteStatement).toHaveBeenCalledWith(expect.objectContaining({
-      Sql: `CREATE TABLE ${tableNamePrefix}${requestIdTruncated} (col1 varchar(1)) DISTSTYLE KEY DISTKEY(col1)`,
+      Sql: `CREATE TABLE ${tableNamePrefix}${requestIdTruncated} (col1 varchar(1) ) DISTSTYLE KEY DISTKEY(col1)`,
```

Comment:
ah

I see.

I hate to do this, but can you revert that change to the spacing I asked in the first round? I hadn't expected it to result in extra spaces everywhere.

Reply:
lol, no problem. It was the last commit anyways, so I just reverted it.